internal/engine/interpreter/interpreter.go

```diff
@@ -848,6 +848,21 @@ func (ce *callEngine) callNativeFunc(ctx context.Context, callCtx *wasm.CallCont
 	bodyLen := uint64(len(frame.f.body))
 	for frame.pc < bodyLen {
 		op := frame.f.body[frame.pc]
+
+		var checkPc bool
+		var prevPc uint64
```

Comment:
I strongly believe that these vars are optimized out as well if unused if IsTest == false